### PR TITLE
releases: migrate UNRELEASED_CONFIG jobs to generated

### PR DESCRIPTION
**Pull request by AlexNPavel**
This PR migrates all pre-4.7 jobs that use `UNRESOLVED_CONFIG` to
jobs generated by `ci-operator`. This is the migration mapping from
the `release-job-migrator` tool:
```
The following jobs have been replaced:
release-openshift-origin-installer-e2e-aws-upgrade-4.4-nightly-to-4.4-nightly -> periodic-ci-openshift-release-master-origin-4.4-nightly-to-4.4-nightly-e2e-aws-upgrade
release-openshift-origin-installer-e2e-aws-upgrade-4.4-stable-to-4.4-ci -> periodic-ci-openshift-release-master-origin-4.4-stable-to-4.4-ci-e2e-aws-upgrade
release-openshift-origin-installer-e2e-aws-upgrade-4.4-stable-to-4.5-ci -> periodic-ci-openshift-release-master-origin-4.4-stable-to-4.5-ci-e2e-aws-upgrade
release-openshift-origin-installer-e2e-aws-upgrade-4.5-stable-to-4.6-ci -> periodic-ci-openshift-release-master-origin-4.5-stable-to-4.6-ci-e2e-aws-upgrade
release-openshift-origin-installer-e2e-azure-upgrade-4.4 -> periodic-ci-openshift-release-master-origin-4.4-e2e-azure-upgrade
release-openshift-origin-installer-e2e-azure-upgrade-4.4-stable-to-4.5-ci -> periodic-ci-openshift-release-master-origin-4.4-stable-to-4.5-ci-e2e-azure-upgrade
release-openshift-origin-installer-e2e-azure-upgrade-4.5 -> periodic-ci-openshift-release-master-origin-4.5-e2e-azure-upgrade
release-openshift-origin-installer-e2e-azure-upgrade-4.5-stable-to-4.6-ci -> periodic-ci-openshift-release-master-origin-4.5-stable-to-4.6-ci-e2e-azure-upgrade
release-openshift-origin-installer-e2e-azure-upgrade-4.6 -> periodic-ci-openshift-release-master-origin-4.6-e2e-azure-upgrade
release-openshift-origin-installer-e2e-gcp-upgrade-4.4 -> periodic-ci-openshift-release-master-origin-4.4-e2e-gcp-upgrade
release-openshift-origin-installer-e2e-gcp-upgrade-4.4-stable-to-4.5-ci -> periodic-ci-openshift-release-master-origin-4.4-stable-to-4.5-ci-e2e-gcp-upgrade
release-openshift-origin-installer-e2e-gcp-upgrade-4.5 -> periodic-ci-openshift-release-master-origin-4.5-e2e-gcp-upgrade
release-openshift-origin-installer-e2e-gcp-upgrade-4.5-stable-to-4.6-ci -> periodic-ci-openshift-release-master-origin-4.5-stable-to-4.6-ci-e2e-gcp-upgrade
release-openshift-origin-installer-e2e-gcp-upgrade-4.6 -> periodic-ci-openshift-release-master-origin-4.6-e2e-gcp-upgrade
```

/hold

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__origin-4.4-nightly-to-4.4-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+base_images:
+  base:
+    name: "4.4"
+    namespace: ocp
+    tag: base
```
Comment (petr-muller) [inline ci-operator/config/openshift/release/openshift-release-master__origin-4.4-nightly-to-4.4-nightly.yaml:5]: Huh are these actually needed?

**Review by petr-muller — COMMENTED** [review]
Where is the `__origin-` part of the filename coming from? Sounds like an opportunity to get rid of some cruft.

**Comment by AlexNPavel** [comment]
> Where is the `__origin-` part of the filename coming from? Sounds like an opportunity to get rid of some cruft.

The `origin` part of the variant refers to (or should refer to) the stream we are using for the `latest` release, the same as the old release job naming. The ocp ci stream is `origin`, the ocp nightly stream is `ocp`, and the okd stream is `okd`. Release jobs that do upgrades almost always use the origin stream. I think calling the ocp ci stream `origin` when okd should technically be origin is a bit confusing, but TRT would like to keep the current naming scheme for the streams.

One of the exceptions to the naming is the 4.4 nightly upgrade. It should probably be `ocp` since we are targeting a nightly as latest, but since we want to make the mapping between the old and new names deterministic and the original called itself `origin-installer`, the name remains as `origin`.

The only release jobs using `UNRESOLVED_CONFIG` at the moment are upgrade jobs, so everything is origin, but other migrations in the future will also have `ocp` and `okd` prefixed variants.

File: ci-operator/config/openshift/release/openshift-release-master__origin-4.4-nightly-to-4.4-nightly.yaml (hunk 1 above)
Comment (AlexNPavel) [reply ci-operator/config/openshift/release/openshift-release-master__origin-4.4-nightly-to-4.4-nightly.yaml:5]: I can check, but IIRC a lot of steps still use `base` as their image (even though it's usually unnecessary and should set to a static `centos:8` or similar) so it needs to be set as a `base_image`.

**Pull request merged**